src/main/java/core/basesyntax/Application.java

```diff
@@ -1,7 +1,14 @@
 package core.basesyntax;
 
+import ball.characteristics.Ball;
+
 public class Application {
     public static void main(String[] args) {
         // create three balls using class Lottery and print information about them in console
+        for (int i = 0; i < 3; i++) {
+            Lottery lottery = new Lottery();
+            Ball ball = new Ball(lottery.getRandomBall());
+            System.out.println(ball.toString());
```

Comment:
you can remove .toString() from println, it's called under the hood anyway

Reply:
yes, but it make up a sentence at once

Reply:
Sorry I didn't really get your question :) You should override toString in the Ball.java class, but when you put something in system out println, .toString of this object will be called